OlenaNehodenko/HW5/HW5.py

```diff
@@ -1,5 +1,25 @@
 #First task: Create a list
-List=[1,2,3,4,5,6]
+num_list=int(input("Enter amount of list:"))
```

Comment:
look at this code
num_list=int(input("Enter amount of list:"))
List=[0]
while len(List)<=num_list:
      List.append(float(List[-1]+1))
print(List)

print("-----------------------------")